src/Inventory/Asset/MainAsset.php

```diff
@@ -754,6 +764,11 @@ public function rulepassed($items_id, $itemtype, $rules_id, $ports_id = 0)
         }
         $val->entities_id = $entities_id;
 
+
+        // set entities_id in known links
```

Comment:
Seems useless; the first add is not under condition, and `$entities_id` is not changed after.

Reply:
removed